packages/vc-verification/src/resolver/Ipfs-credential-resolver.ts

```diff
@@ -81,17 +86,29 @@ export class IpfsCredentialResolver implements CredentialResolver {
    *  provider,
    *  registrySettings,
    *  didStore );
-   * const credential = credentialResolver.getVerifiableCredential('did:ethr:1234', 'sampleRole');
+   * const credential = credentialResolver.getVerifiableCredential('did:ethr:1234', 'sampleRole', roleCredentialCache);
    * ```
    *
    * @param did subject DID for which the credential needs to be fetched
    * @param namespace role for which the credential needs to be fetched
+   * @param roleCredentialCache Cache to store role credentials
```

Comment:
I think it would be good to mention that the cache is populated (to at least communicate the side-effect)
```suggestion
   * @param roleCredentialCache Cache to store role credentials. Cache is updated with all credentials retrieved for the DID
```